packages/kit/src/exports/public.d.ts

```diff
@@ -1012,6 +1012,10 @@ export interface RequestEvent<
 	 * related to the data request in this case. Use this property instead if the distinction is important to you.
 	 */
 	isDataRequest: boolean;
+	/**
+	 * `true` if the request comes from a call to one of our own API endpoints made during SSR (or prerendering). SvelteKit in this case directly calls `+server.js` without the overhead of actually making an HTTP request.
```

Comment:
I don't think the description is quite right - this also happens if you call this from the client via "REST" if it's a `+page.server.js`. My attempt at a better description:

```suggestion
	 * `true` for `+server.js` calls coming from SvelteKit without the overhead of actually making an HTTP request. This happens when you make relative fetch requests on the server.
```

Reply:
Yeah, that's a good call. It's updated the docs.